Description: Hardcode frobby version.
 The header stdinc.h isn't included in Debian's frobby package, so we
 can't get the version that way.  Just hardcode it for now, since the
 version isn't likely to increase any time soon.
 Maybe one day, we could get it from apt:
 apt-cache show frobby | grep "^Version" | sed 's/Version: \(.*\)-.*/\1/'
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-13

--- a/M2/Macaulay2/d/version.dd
+++ b/M2/Macaulay2/d/version.dd
@@ -40,7 +40,6 @@
    #include <mathicgb.h>
    #define stringize0(a) #a
    #define stringize(a) stringize0(a)
-   #include <stdinc.h> /* this is part of frobby, amazingly */
    extern char timestamp[];
    #undef ASSERT
    #ifdef HAVE_SCSCP
@@ -212,7 +211,7 @@
 	#endif
 	"),
    "ntl version" => Ccode(constcharstar,"NTL_VERSION"),
-   "frobby version" => Ccode(constcharstar,"constants::version"),
+   "frobby version" => "0.9.0",
    "flint version" => Ccode(constcharstar,"
 	 #if HAVE_FLINT
 	   FLINT_VERSION
